src/rpc/RPCHelpers.cpp

```diff
@@ -709,22 +670,21 @@ traverseOwnedNodes(
         auto hintDir =
             backend.fetchLedgerObject(hintIndex.key, sequence, yield);
 
-        if (hintDir)
-        {
-            ripple::SerialIter it{hintDir->data(), hintDir->size()};
-            ripple::SLE sle{it, hintIndex.key};
+        if (!hintDir)
+            return Status(ripple::rpcINVALID_PARAMS, "Invalid marker");
```

Comment:
Is this rippled's behavior? I think this corresponds to passing a marker that has a bogus hint, which is the single integer at the end of the marker.